transformer.py

```diff
@@ -146,30 +148,44 @@ def perform_process(transformer: transformer_class.Transformer, check_md: dict,
     result = {}
     file_md = []
 
-    file_list = os.listdir(check_md['working_folder'])
+    file_list = check_md['list_files']()
 
     # Extract necessary parameters from metadata
-    scan_distance = float(full_md['sensor_variable_metadata']['scan_distance_mm'])/1000.0
-    scan_direction = int(full_md['sensor_variable_metadata']['scan_direction'])
-    point_cloud_origin = full_md['sensor_variable_metadata']['point_cloud_origin_m']['east']
+    for one_metadata in full_md:
+        if 'sensor_variable_metadata' in one_metadata:
+            scan_distance = float(one_metadata['sensor_variable_metadata']['scan_distance_mm'])/1000.0
```

Comment:
Won't scan_distance_mm/ 1000 return a value with units of nm? What are the units here? even if it was already here, unidentified magic numbers are scary!

Reply:
It was there already and I have no idea what it represents. I agree about magic numbers being scary. I will try to find out what the `1000.0` represents

Reply:
Maybe @max-zilla knows?

Reply:
He believes it's a conversion to Meters. I will make the change